packages/docs/tutorial/tutorial-actions-operators.yaml

```diff
@@ -228,7 +211,7 @@ _ref:
 
             ## Hiding blocks
 
-            We would like to ask users if they restarted their device if they want to create a bug report. To do this we will add a input block with a `visible` property. This property expects a boolean value, so we will use the [`_eq`](/_eq) [`operator`](/operators) to test if the ticket type is equal to `Bug report`
+            We would like to ask user if they had purchased the product that they are formulating a complaint about, within the last month. To do this we will add a input block with a `visible` property. This property expects a boolean value, so we will use the [`_eq`](/_eq) [`operator`](/operators) to test if the ticket type is equal to `Complaint`
```

Comment:
toa ask the user

Reply:
Change made in 9654eef9a0e0ece57d5e66f6400d8265e9346466